src/sentry/web/frontend/newest_issue.py

```diff
@@ -0,0 +1,38 @@
+from django.http import HttpResponse, HttpResponseRedirect
+from django.urls import reverse
+from rest_framework.request import Request
+
+from sentry.models.group import Group, GroupStatus
+from sentry.web.frontend.base import OrganizationView
+
+
+class NewestIssueView(OrganizationView):
+    def handle(self, request: Request, organization, issue_type="error", **kwargs) -> HttpResponse:
+        issue_list_url = organization.absolute_url(
+            reverse("sentry-organization-issue-list", args=[organization.slug])
+        )
+        if issue_type == "error":
+            range = (0, 1000)
+        if issue_type == "performance":
+            range = (1000, 2000)
+        elif issue_type == "profile":
+            range = (2000, 3000)
```

Comment:
Same comment from the other pr, we should use `get_group_types_by_category` here.